Ansible/ansible_collections/jfrog/platform/ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+host_key_checking = false
+stdout_callback = debug
+remote_tmp = /tmp/.ansible/tmp
+private_key_file=~/.ssh/ansible-jfrog.key
```

Comment:
Seems wrong that we are hard coding the location of a private key like this? 

Can we explain why this is here?

Shouldn't the customer generate this key and give the location?

Reply:
Yes, will be fixed in next version